### ISSUE TITLE
nan loss encountered

### ISSUE THREAD

**Issue by jiaxiaolei007** [allenai/OLMo#498]
### ❓ The question

During the training process, an error message "nan loss encoded" occurred. Upon checking, it was found that the ce_batch_loss value had "nan". What is this ce_batch_loss and how is this situation caused? How should we solve it?

**Comment by 2015aroras** [comment]
`ce_batch_loss` is 'cross entropy batch loss', the cross entropy loss summed over a batch. There are many possible causes for nan loss. For example, the training loss could have been diverging towards infinity (e.g., because the learning rate was too high) or a division by 0 might have happened somewhere. You'll probably need to dive deeper to find the cause of the nan.

**Comment by dumitrac** [comment]
@jiaxiaolei007 - please reopen this item or a new one if you have further questions.
